index.html

```diff
@@ -0,0 +1,224 @@
+        <div class="techstack-container">
+          <h3>css</h3>
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
+      </div>
+
+      <div class="work">
+        <h2>Profesional Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div class="techstack-container">
+          <h3>html</h3>
+          <h3>bootstrap</h3>
+          <h3>Ruby</h3>
+        </div>
+        <button class="project-button">See Project</button>
```

Comment:
- Kindly make sure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") is a list `ul` and each of them a `li` that for a semantic reason :+1: